src/tigerbeetle/cli.zig

```diff
@@ -42,21 +42,28 @@ const CliArgs = union(enum) {
     },
 
     start: struct {
+        // Stable CLI interface.
         addresses: []const u8,
-        limit_storage: flags.ByteSize = .{ .value = constants.storage_size_limit_max },
+        cache_grid: ?flags.ByteSize = null,
+        positional: struct {
+            path: [:0]const u8,
+        },
+
+        development: bool = false,
```

Comment:
Move `development` before `positional`? Or is there some reason why this one is somewhat separate?

Reply:
No strong reason, when I was writing it out I was thinking in terms of what disables multiversion upgrading.